app/src/main/java/com/onboarding/parkingsystemkotlin/adapter/ReservationAdapter.kt

```diff
@@ -11,18 +13,21 @@ class ReservationAdapter(private val list: List<Reservation>) : RecyclerView.Ada
     override fun getItemCount() = list.size
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
-        return ViewHolder(ItemReservationBinding.inflate(LayoutInflater.from(parent.context), parent, false))
+        return ViewHolder(ItemReservationBinding.inflate(LayoutInflater.from(parent.context), parent, false), parent.context)
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         holder.bind(list[position])
     }
 
-    class ViewHolder(private val binding: ItemReservationBinding) : RecyclerView.ViewHolder(binding.root) {
+    class ViewHolder(private val binding: ItemReservationBinding, private val context: Context) : RecyclerView.ViewHolder(binding.root) {
         fun bind(reservation: Reservation) {
-            binding.tvItemReservationParkingLot.text = reservation.getParkingLot().toString()
-            binding.tvItemReservationStartDate.text = reservation.getStartDateString()
-            binding.tvItemReservationEndDate.text = reservation.getEndDateString()
+            binding.textViewItemReservation.text = context.getString(
+                R.string.reservation_item_text_view,
+                reservation.getParkingLot(),
+                reservation.getStartDateString(),
+                reservation.getEndDateString()
+            )
```

Comment:
Mmm, you could use differents view's to display each field value. for example; 
PARKING SLOT : $parkingSlorValue
START DATE: $starSateValue...
try to improve pls

Reply:
done

Reply:
done